src/chapters/style/documentation.typ

```diff
@@ -0,0 +1,247 @@
+#import "/src/util.typ": *
+#import mantys: *
+
+= Documentation <sec:style:docs>
+#wbox[
+  This section documents the syntax of documentation commens, see @sec:api:docs for the contents and purpose of documentation comments.
+]
+
+Documentation is placed on special comments using three forward slashes `///` and an optional space.
```

Comment:
Maybe we could start off the documentation with a short but real-world example of how it's done right (also with one or two parameters and return value)? Everything that comes after that will be much easier to follow then